src/components/informationCard/InformationCard.css

```diff
@@ -0,0 +1,32 @@
+.infoCard {
+    flex: 0 1 calc(50% - 30px);
+    background-color: var(--color-background-light);
+    border-radius: 20px;
+    border: none;
+    padding: 30px;
+    position: relative;
+}
+
+.info-card-title {
+    font-size: 35px;
+    font-weight: 400;
+}
+
+.info-card-content {
+    line-height: 2.5;
+}
+
+.info-card-experience-tag {
+    line-height: 2.5;
+}
+
+.arrow-button {
+    position: absolute;
+    right: 15px;
+    bottom: 15px;
+    padding: 15px;
+    margin: 2px;
```

Comment:
Je hebt de knop al absoluut gepositioneerd, dus om dan ook `margin` toe te voegen is een beetje krom! Als je die 2 pixels extra ruimte wil, kun je de offset beter verhogen van 15px naar 17px :)